src/auto-evo/simulation/food_source/EnvironmentalFoodSource.cs

```diff
@@ -15,14 +15,16 @@ public EnvironmentalFoodSource(Patch patch, Compound compound, float foodCapacit
 
             this.patch = patch;
             this.compound = compound;
-            totalEnvironmentalEnergySource = patch.Biome.Compounds[this.compound].Ambient * foodCapacityMultiplier;
+            totalEnvironmentalEnergySource = compound.InternalName == "sunlight" ?
```

Comment:
Why is sunlight a special case here? This doesn't make sense to me as if we average the values for sunlight to track how well species can survive, should we not also average all the other compounds as well, because in the future we might model like oxygen levels dropping or something like that during the night. And the same exact logic with the averages should apply to that.